ci/test-conformance-eks.sh

```diff
@@ -270,10 +270,18 @@ function deliver_antrea_to_eks() {
     # because they might be being used in other builds running simultaneously.
     docker image prune -f --filter "until=2h" > /dev/null
     docker system df -v
+    free_space=$(df -h -B 1G / | awk 'NR==2 {print $4}')
+    free_space_threshold=40
+    if [[ $free_space -lt $free_space_threshold ]]; then
+        # If cleaning up dangling images unused in the last hour doesn't free up sufficient disk space,
+        # we will have to clean up all buildkit cache to release enough disk space.
+        docker buildx prune -af > /dev/null
+    fi
+    docker buildx du
     set -e
```

Comment:
There are code duplication, I feel it's better to move them into a shell file, maybe docker-utils.sh?

Reply:
Added it to utils.sh.